src/build.rs

```diff
@@ -11,10 +11,30 @@ pub fn rustup_add_wasm_target() -> Result<(), Error> {
         emoji::TARGET
     );
     let pb = PBAR.message(&step);
+    let nightly_check = Command::new("rustc").arg("+nightly").arg("-V").output()?;
+    if !nightly_check.status.success() {
+        let sub_bar = PBAR.message(&format!(
```

Comment:
There's another PR where this is going to change a little so be prepared to rebase!

Specifically here:
https://github.com/ashleygwilliams/wasm-pack/pull/151

You'll need to add this function to the vec that way it knows to call it and how to calculate how many steps there are etc.

Reply:
Let me know when I need to perform the rebase